Src/drivetrain_controller.c

```diff
@@ -0,0 +1,69 @@
+//
+// Created by hunter on 11/15/19.
+//
+
+#include "drivetrain_controller.h"
+#include <string.h>
+#include <FreeRTOS.h>
+#include <can_manager.h>
+#include <rt_conf.h>
+#include <task.h>
+#include "VESC.h"
+#include "stdlib.h"
+
+//I am telling the motor controllers to implement a twist message until I get another twist message
+double new_speed_left = 0;
+double new_speed_right = 0;
+
+
+void drivetrain_move(rmc_can_msg msg) {
+    // Because of the mask we only get messages that have our ID
+    int32_t cmd_speed = 0; //in mm/s
+    int32_t cmd_angV = 0;
+
+    if(msg.id & 0xFF != DRIVETRAIN_SYS_ID)
```

Comment:
There's no need to compare against the drivetrain ID because this is done already in the CAN dispatcher.